database/migrations/2017_04_17_045658_create_orders_table.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateOrdersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('orders', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('title');
+            $table->text('description');
+            $table->float('weight')->default(0);
+            $table->float('price_bid')->default(0);
+            $table->string('reciver_name');
+            $table->string('reciver_address');
+            $table->string('reciver_phone');
+            $table->string('lnglat_start');
```

Comment:
[rank B] I think these coordinates should be separate as independency field: latitude and longitude